spec/features/restaurants/visitor_tries_update_restaurant_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'rails_helper'
+
+feature 'prevent unuathorized updates to restaurant' do
+  context 'as a visitor' do
+    scenario 'I cannot save a restaurant' do
+      visit new_restaurant_path
+      expect(page).to have_content("Log in")
+    end
+
+    scenario 'I cannot edit a restaurant' do
+      visit edit_restaurant_path("1")
+      expect(page).to have_content("Log in")
+    end
```

Comment:
This test is confusing as well. I should expect to not see content "Edit" or something like that.